ee/clickhouse/queries/funnels/funnel_correlation.py

```diff
@@ -521,17 +538,101 @@ def _run(self) -> Tuple[List[EventOddsRatio], bool]:
         events = positively_correlated_events[:10] + negatively_correlated_events[:10]
         return events, skewed_totals
 
+    def construct_people_url(self, success: bool, event_definition: EventDefinition) -> Optional[str]:
+        #  NOTE: initially only supporting events type, and there is an implicit
+        #  events type when correlation_type is not set.
+        if not self._filter.correlation_type or self._filter.correlation_type == FunnelCorrelationType.EVENTS:
+            # NOTE: we need to convert certain params to strings. I don't think this
+            # class should need to know these details. As we are trying to encode
+            # the filter dict into form encoding, but the persons endpoint is
+            # expecting some attributes to be further encoded as JSON, e.g.
+            # `funnel_correlation_person_entity`, `entities`.
+            # NOTE: I tried using `self._filter.with_data` but this caused errors to
+            # be raised either in `to_dict` or when deserializing on the people endpoint
+            # TODO: remove url serialization details from this class, it likely
+            # belongs in the application layer, or perhaps `FunnelCorrelationPeople`
+            params = self._filter.to_dict()
+            params = {
+                **params,
+                "events": json.dumps(params["events"]),
+                "funnel_correlation_person_converted": "true" if success else "false",
+                "funnel_correlation_person_entity": json.dumps(
+                    {"id": event_definition["event"], "type": "events", "properties": event_definition["properties"]}
+                ),
+            }
+            return f"/api/person/funnel/correlation/?{urllib.parse.urlencode(params)}"
+
+        if self._filter.correlation_type == FunnelCorrelationType.EVENT_WITH_PROPERTIES:
+            event_name, property_name, property_value = event_definition["event"].split("::")
+            params = self._filter.to_dict()
+            params = {
+                **params,
+                "events": json.dumps(params["events"]),
+                "funnel_correlation_event_names": json.dumps(params["funnel_correlation_event_names"]),
+                "funnel_correlation_person_converted": "true" if success else "false",
+                "funnel_correlation_person_entity": json.dumps(
+                    {
+                        "id": event_name,
+                        "type": "events",
+                        "properties": [
+                            {"key": property_name, "value": property_value, "type": "event", "operator": "exact"}
```

Comment:
As you mentioned earlier, this is incorrect for `$autocapture` events. I'm okay with this if you want to keep a separate path in the frontend to get autocapture persons (i.e. keep it like it is now)

But, if you want to move to this, I think we'd have to handle autocapture correctly here?